atlas-starter.py

```diff
@@ -0,0 +1,91 @@
+import pymongo
+
+# Replace the placeholder data with your Atlas connection string. Be sure it includes
+# a valid username and password! Note that in a production environment,
+# you should not store your password in plain-text here.
+
+client = pymongo.MongoClient(
+      "mongodb+srv://<username>:<password>@<cluster-name>/test?retryWrites=true&w=majority")
+
+# use a database named "my_database"
+
+db = client.my_database
```

Comment:
What happens if a user tries to connect without replacing the URI, or without filling in their password in the pasted URI? In some of our other examples, we've tried to output something (slightly) more helpful so users can debug their issues.

Reply:
Another small nit: the other examples use "myDatabase" here. Even if that's not pythonic, could we change the database name so that each example is consistent?